synapse/models/person.py

```diff
@@ -43,6 +45,8 @@ def getModelDefs(self):
 
                 ('ps:contact', ('guid', {}), {
                     'doc': 'A GUID for a contact info record.',
+                    'interfaces': ('phys:object',),
```

Comment:
I'm not clear on the benefits of having phys:object linked to ps:contact - in the case of a person, would I use this to capture something like the info on my ID? (As of this date, I lived at X, and was 5 ft 3 inches and 8 million lbs)? Or would I use phys:object to capture my house at ps:contact:address?